bitnami/airflow/templates/_git.tpl

```diff
@@ -0,0 +1,174 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I'm not aware of us having any other helpers file apart from the common chart, than `_helpers.tpl`. Could you rename it so it is easier to understand this file's purpose? I.e. `_git_helpers.tpl`.